android-components/components/browser/state/src/main/java/mozilla/components/browser/state/reducer/TranslationsStateReducer.kt

```diff
@@ -176,6 +187,15 @@ internal object TranslationsStateReducer {
                     }
                 }
 
+                TranslationOperation.FETCH_DOWNLOADED_LANGUAGES -> {
+                    state.copy(
```

Comment:
Recommend only setting the session store error on this action:

```
                    state.copyWithTranslationsState(action.tabId) {
                        it.copy(
                            translationError = action.translationError,
                        )
                    }
```

I'm leaning towards thinking we should avoid `downloadedLanguages = null,`, in case another tab was successful on the global state.

General design thought is particular tabs call the `TranslationOperation` when something goes wrong and `Init` goes its own path globally.

Reply:
updated to session error